README.md

```diff
@@ -70,10 +70,9 @@ GPT-NeoX parameters are defined in a YAML configuration file which is passed to
 
 All functionality follows the pattern `./deepy.py main_function.py -d configs small.yml local_configs.yml`
 We currently offer four main functions:
-1. `train.py` is used for training and finetuning models.
+1. `train.py` is used for training, finetuning, and distilling models.
```

Comment:
it's not used for distilling yet though?

Reply:
Fair point. Removed and directed people to the `distill-gpt-neox` branch in the Distilling section.